scrape/scrape_test.go

```diff
@@ -550,12 +550,12 @@ func TestScrapeLoopRun(t *testing.T) {
 	select {
 	case <-signal:
 	case <-time.After(5 * time.Second):
-		t.Fatalf("Cancelation during initial offset failed")
+		t.Fatalf("Cancellation during initial offset failed")
```

Comment:
A quick google search tells me that `cancelation` is also fine.

Reply:
That's Americans forgetting their own spellings, it's meant to be two l's.

Reply:
Oh that debate again… :o) One day, we might need a formal vote of prometheus-team to solve it once and forever.

Personally, my proposal would be to ban flip-flopping between the two spellings in PRs so that we are not lured into wasting time on it ever again. Both spellings exist, and we have other problems than deciding for the world which one to use. There isn't even any ambiguity involved (like with Oxford comma or "Prometheus'" vs. "Prometheus's").

For the interested: https://en.wiktionary.org/wiki/cancelation probably summarizes it neatly.

Reply:
Ah, I'd not seen that it was one of those explicit US changes.